eventual-cassandra/src/test/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/ReplicatedCassandraTest.scala

```diff
@@ -104,6 +104,7 @@ class ReplicatedCassandraTest extends AnyFunSuite with Matchers {
 
       val expected = State(
         actions = List(
+          Action.DeleteRecords(key, SegmentNr.journal(SeqNr.unsafe(1), segmentSize)),
```

Comment:
Please elaborate why this action happened now but not before the change

Reply:
It was never recorded before - I added it here: 
https://github.com/evolution-gaming/kafka-journal/pull/682/files/32e7a11459e1c8720988033054639e53ce19ee64#diff-29aefcb72c7b9fba4c21d3237b491940e36217107a2b98c2ed8a3d2c4674c7b0R1885

Similar for deletion of record in `metajournal` table